### PR TITLE
Improve E0133 error explanation

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @steveklabnik

### CODE REVIEW

File: src/librustc/diagnostics.rs [hunk 1]
```diff
@@ -364,6 +364,18 @@ type X = u32; // ok!
 "##,
 
 E0133: r##"
+Unsafe code was used outside unsafe function or block.
```
Comment (steveklabnik) [inline src/librustc/diagnostics.rs]: "outside of an unsafe function"

**Comment by GuillaumeGomez** [comment]
@ bors: r+ rollup

**Pull request merged**